expyfun/io/_parse.py

```diff
@@ -4,6 +4,40 @@
 
 import numpy as np
 import csv
+import ast
+import json
+
+
+def read_tab_raw(fname):
+    """Read .tab file from expyfun output without segmenting into trials
```

Comment:
PEP257 says we should. I noticed that some/most of the other functions here had them so I figured we might as well.